File: Linguagens_Programação/C/4.c
/*4. Escreva um programa que converta temperaturas de Fahrenheit para Celsus,
seguindo a equação 𝐶 = (𝐹−32)×5/9*/
#include <stdio.h>
int main()
{
    int temp_F,temp_C;

    printf("Digite uma temperatura em Fahrenheit: ");
    scanf("%i",&temp_F);
   
    temp_C = ((temp_F-32)*5)/9;
    printf("A temperatura %i em Fahrenheit equivale a %i graus em Celsius!",temp_F,temp_C);

    return 0;
}

File: Linguagens_Programação/Go/4.go
/*4. Escreva um programa que converta temperaturas de Fahrenheit para Celsius, 
seguindo a equação 𝐶 = (𝐹−32)×5/9. */
package main
import "fmt"

func main() {
    var F,C int

    fmt.Println("Digite uma temperatura em Fahrenheit: ")
    fmt.Scanln(&F)
    
    C = (F - 32)*5 / 9
    fmt.Printf("O valor de %d graus em Fahrenheit equivale a %d graus em Celsius",F,C)
}

